controller/registry/private.py

```diff
@@ -87,8 +87,7 @@ def _put_first_image(repository_path):
 
 
 def _api_call(endpoint, data=None, headers={}, cookies=None, request_type='GET'):
-    # FIXME: update API calls for docker 0.10.0+
-    base_headers = {'user-agent': 'docker/0.9.0'}
+    base_headers = {'user-agent': 'docker/0.10.1'}
```

Comment:
It should act the same, it's just more or less compatibility concerns. I'm just thinking that since they dropped 0.9.0 client support, they might drop 0.10.1 support in the future. Being compatible with docker/1.0.0 would ensure that we wouldn't have to fix this in the future, since I'm assuming that they wouldn't drop v1 api support in the near future.